src/framework/application.js

```diff
@@ -162,6 +162,8 @@ pc.extend(pc, function () {
         this.renderer = new pc.ForwardRenderer(this.graphicsDevice);
         this.lightmapper = new pc.Lightmapper(this.graphicsDevice, this.root, this.scene, this.renderer, this.assets);
         this.once('prerender', this._firstBake, this);
+        this.batcher = new pc.BatchManager(this.graphicsDevice, this.root, this.scene);
+        this.once('prerender', this._firstBatch, this);

```

Comment:
You have the same line at line 164 above - is that on purpose? I'm guessing you only want _firstBatch to be called once but you subscribe to the prerender event twice.

Reply:
164 is firstBake not batch ;)

Reply:
Oops